### PR TITLE
DOI & Tool linking syntax Update Infinium Human Methylation BeadChip Training

### PR DESCRIPTION
**Pull request by kkamieniecka**
Fixing outdated syntax of Infinium Human Methylation BeadChip Training

### CODE REVIEW

**Comment by kkamieniecka** [comment]
@hexylena I think this one is ready to be merged (fixed syntax and DOI)

File: topics/epigenetics/tutorials/ewas-suite/tutorial.md [hunk 1]
```diff
@@ -127,7 +133,7 @@ However, for the purpose of this tutorial we would like you to upload phenotype
 
 > <hands-on-title>Import `phenotypeTable.txt` from [Zenodo](https://zenodo.org/record/1251211#.WwREQ1Mvz-Y) or data library:</hands-on-title>
 >    ```
```
Comment (hexylena) [inline topics/epigenetics/tutorials/ewas-suite/tutorial.md:135]:
```suggestion
> ```
```

File: topics/epigenetics/tutorials/ewas-suite/tutorial.md [hunk 2]
```diff
@@ -127,7 +133,7 @@ However, for the purpose of this tutorial we would like you to upload phenotype
...
->    https://zenodo.org/record/1251211/files/phenotypeTable.txt
+> https://zenodo.org/record/1251211/files/phenotypeTable.txt
 >    ```
```
Comment (hexylena) [inline topics/epigenetics/tutorials/ewas-suite/tutorial.md:137]:
```suggestion
> ```
```

**Comment by hexylena** [comment]
Should we (at some point?) merge the ELIXIR-UK-DaSH branch into main on the training material side? https://github.com/galaxyproject/training-material/tree/ELIXIR-UK-DaSH

**Pull request merged**